src/stats.rs

```diff
@@ -26,6 +26,34 @@ impl Ewma {
     }
 }
 
+/// Exponentially Weighted Moving Average that starts with a default average value
+#[allow(dead_code)]
+#[derive(Clone, Copy, Debug)]
+pub struct EwmaDefault {
+    average: f64,
+    alpha: f64,
+}
+
+#[allow(dead_code)]
+impl EwmaDefault {
```

Comment:
I wonder if the ARC code could be adapted to use this Ewma instead of having to deal with the `Option` for the average. That's work for another issue, though.